Index: mynewFile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>hello\r\n我在idea进行了编辑\r\n远程第二分支编辑\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- mynewFile	(revision 24d64a7c8a631ed02dcbfba54d0b04ca01c96fa1)
+++ mynewFile	(date 1633104766608)
@@ -1,3 +1,3 @@
 hello
 我在idea进行了编辑
-远程第二分支编辑
+远程第一分支编辑
